stdlib/ballerina-builtin/src/main/ballerina/util/natives.bal

```
@@ -47,3 +47,8 @@ public native function base16ToBase64Encode (string baseString) returns (string)
 @Param {value:"s: string to be encoded"}
 @Return {value:"the encoded string."}
 public native function base64ToBase16Encode (string baseString) returns (string);
+
+@Description {value:"Parse JSON string to generate JSON object."}
+@Param {value:"s: JSON string"}
+@Return {value:"JSON object."}
+public native function parseJson (string s) returns (json);
```

Shouldn't we return an error if we cannot parse the string?